### PR TITLE
Fix favorites index

### PR DESCRIPTION
**Pull request by jjennifergu**
## Overview

Quick fix on index out of bounds error on Favorites carousel

### CODE REVIEW

**Review by antoinztte — APPROVED** [review]
🔥

File: .DS_Store
Comment (NgaiJustin) [inline .DS_Store:1]: Add this to the `.gitignore`

File: Eatery Blue/Eatery Blue/UI/HomeScreen/HomeModelController.swift [hunk 1]
```diff
@@ -92,7 +92,7 @@ class HomeModelController: HomeViewController {
         carouselView.layoutMargins = UIEdgeInsets(top: 0, left: 16, bottom: 0, right: 16)
         carouselView.titleLabel.text = title
 
-        for i in 0...2 {
+        for i in 0..<min(carouselEateries.count,3) {
```
Comment (NgaiJustin) [inline Eatery Blue/Eatery Blue/UI/HomeScreen/HomeModelController.swift:95]: Is there a reason why this isn't just `carouselEateries.count`?

File: Eatery Blue/Eatery Blue/.DS_Store
Comment (NgaiJustin) [inline Eatery Blue/Eatery Blue/.DS_Store:1]: Same with this. If you add `.DS_Store` to the `.gitignore` it should ignore all `.DS_Store` in all directories. If you need to you can also run `git rm --cached .DS_Store` to manually remove the `DS_Store` files that were already added to the repo

**Review by NgaiJustin — APPROVED** [review]
🚀

**Review by ReadeP — APPROVED** [review]

File: Eatery Blue/Eatery Blue/UI/HomeScreen/HomeModelController.swift (hunk 1 above)
Comment (jjennifergu) [reply Eatery Blue/Eatery Blue/UI/HomeScreen/HomeModelController.swift:95]: Yea because we only want to show max 3 eateries in the carousel

**Pull request merged**